spring-kafka/src/main/java/org/springframework/kafka/retrytopic/RetryTopicConfigurationBuilder.java

```diff
@@ -241,6 +244,41 @@ public RetryTopicConfigurationBuilder setTopicSuffixingStrategy(TopicSuffixingSt
 		return this;
 	}
 
+	/**
+	 * Configure the {@link SameIntervalTopicReuseStrategy}.
+	 *
+	 * <p>Note: for fixed backoffs, when this is configured as
+	 * {@link SameIntervalTopicReuseStrategy#SINGLE_TOPIC}, it has precedence over
+	 * the configuration done through
+	 * {@link #useSingleTopicForFixedDelays(FixedDelayStrategy)}.
+	 * @param sameIntervalTopicReuseStrategy the strategy.
+	 * @return the builder.
+	 * @since 3.0.4
+	 */
+	public RetryTopicConfigurationBuilder sameIntervalTopicReuseStrategy(SameIntervalTopicReuseStrategy sameIntervalTopicReuseStrategy) {
+		this.sameIntervalTopicReuseStrategy = sameIntervalTopicReuseStrategy;
+		return this;
+	}
+
+	/**
+	 * Configure the use of a single retry topic
+	 * for the attempts that have the same backoff interval
+	 * (as long as these attempts are in the end of the chain).
+	 *
+	 * Currently used only for the last retries of exponential backoff,
+	 * and in a future release this will dictate whether to use
+	 * a single retry topic for fixed backoff.
```

Comment:
@jgslima Looks like we missed cleaning this up - I just tested it and the `SameIntervalTopicReuseStrategy` works fine for a fixed delay too.

It's true that the other strategy will be removed next release, but they can switch to the new one today (or am I missing something?).

Reply:
You are right. :(
One of the last changes was indeed to allow the new strategy to be used for fixed delays.
My mistake.
Do you want me to open a PR?

Reply:
Thanks for confirming; no; I'll just push a fix.

Thanks again for implementing this.